src/Middleware/SpaServices.Extensions/src/SpaDefaultPageMiddleware.cs

```diff
@@ -24,6 +25,12 @@ public static void Attach(ISpaBuilder spaBuilder)
             // Rewrite all requests to the default page
             app.Use((context, next) =>
             {
+                // If we have an Endpoint, then this is a deferred match - just noop.
+                if (context.GetEndpoint() != null)
+                {
+                    return next();
+                }
+
```

Comment:
I coulnd't find any tests. #YOLO

Reply:
Historically it was E2E tests, but they got removed. So yeah.